src/Generators/templates/taxonomies/config.php

```diff
@@ -5,26 +5,31 @@
 use Tribe\Libs\Taxonomy\Taxonomy_Config;
 
 class Config extends Taxonomy_Config {
-	protected $taxonomy = '%4$s';
-	protected $post_types = [ %5$s ];
+
+	protected string $taxonomy = '%4$s';
+
+	/**
+	 * @var string[]
+	 */
+	protected array $post_types = [ %5$s ];
```

Comment:
The max version for tribe libs at the moment is PHP 7.2 so this should not include property types (only commented)